static/css3/fragment.css

```diff
@@ -131,3 +131,6 @@ hr {
     color:lime;
 }
 */
+[name^="l-"]:target + .tm, [name^="l-"]:target + .tm + .nk, [name^="l-"]:target + .tm + .nka {
+    background-color: #05ffff;
```

Comment:
Could you please amend the most recent commit to introduce one of the lighter palettes from here https://colorswall.com/palette/24612/ ?

The colors listed in that link belong from the #008080 family but the one that you used does not.